packages/api/test/e2e/promise-alex.spec.ts

```diff
@@ -32,30 +34,31 @@ describe.skip('alex queries', () => {
       console.log('api.query.staking.validators():', res.toJSON());
 
       done();
-    });
+    }).then().catch();
   });
 
-  it('retrieves a single value', (done) => {
-    api.query.staking.validators('5DuiZFa184E9iCwbh4WjXYvJ88NHvWJbS8SARY8Ev1YEqrri', (res) => {
-      console.error(res);
-
-      console.log('api.query.staking.validators(id):', res.toJSON());
+  describe('retrieves a single value', () => {
+    it('retrieves the list of stash validators', (done) => {
+      api.query.staking.validators('5DuiZFa184E9iCwbh4WjXYvJ88NHvWJbS8SARY8Ev1YEqrri', (res) => {
+        console.error(res);
+        console.log('api.query.staking.validators(id):', res.toJSON());
 
-      done();
+        done();
+      }).then().catch();
     });
 
     it('Gets the hash of the last finalized header', async (done) => {
       api.rpc.chain.getFinalizedHead((head) => {
         expect(head instanceof Hash).toBe(true);
         done();
-      });
+      }).then().catch();
```

Comment:
.then is useless. Also, we could console.error the .catch just in case

Reply:
Otherwise I'm getting the `Promises must be handled appropriately` error on all promises without `.then().catch()`.

Do you know a better solution?

Reply:
- .catch should be there
- .then can be removed